src/Service/CognitoIdentityProvider/src/CognitoIdentityProviderClient.php

```diff
@@ -233,6 +235,74 @@ public function adminDeleteUser($input): Result
         return new Result($response);
     }
 
+    /**
+     * Disable a user as an administrator. Works on any user.
+     *
+     * @see https://docs.aws.amazon.com/cognito-user-identity-pools/latest/APIReference/API_AdminDisableUser.html
+     * @see https://docs.aws.amazon.com/aws-sdk-php/v3/api/api-cognito-idp-2016-04-18.html#admindisableuser
+     *
+     * @param array{
+     *   UserPoolId: string,
+     *   Username: string,
+     *   @region?: string,
+     * }|AdminDisableUserRequest $input
+     *
+     * @throws ResourceNotFoundException
+     * @throws InvalidParameterException
+     * @throws TooManyRequestsException
+     * @throws NotAuthorizedException
+     * @throws UserNotFoundException
+     * @throws InternalErrorException
+     */
+    public function adminDisableUser($input): Result
```

Comment:
This method should be part of the `manifest.json` file.

Use ```
./generate CognitoIdentityProvider AdminDisableUser
./generate CognitoIdentityProvider AdminEnableUser
```

Also, please add an entry in the CHANGELOG file

Reply:
thank you for your review, i've made some changes to my PR :)